models/public/aclnet/aclnet.md

```diff
@@ -0,0 +1,68 @@
+# aclnet
+
+## Use Case and High-Level Description
+
+The `AclNet` model is designed to perform sound classification.
+The `AclNet` model is trained on a dataset of environmental sounds (DES-53).
+For details about the model, see this [paper](https://arxiv.org/pdf/1811.06669.pdf).
+
+The model input is a segment of PCM audio samples in [N, C, H, W] format with the shape [1, 1, 1, 16000].
```

Comment:
> H, W

That doesn't make sense - audio doesn't have height or width.

Reply:
This nomenclature follows the adopted convention format of NCHW typically used in deep learning frameworks, where the input data is represented in a 2-dimensional matrix of size Height x Width (H x W),  i.e. in the case of raw audio H = 1 and W = 16,000. Of course, strictly speaking, audio signals do not have height and width, but the input data representation following the mentioned convention facilitates the handling of the raw audio as the input data represented by a vector of size H x W = 1 x 16,000 (1 second of audio).

Reply:
I understand what the convention is, but it only makes sense for images. We can pretend that the audio segment is an image of height 1, but that's an extra abstraction that doesn't seem to gain us anything - it just obscures the meanings of the dimensions. IMO, we should call things what they are and describe the format as [N, C, 1, L], where L is the length of the segment (or the number of samples in the segment).

Reply:
I agree that the use of [N,C,1,L] format will avoid further confusion. Additionally, L could be of variable length, not necessarily  a fixed size of 16,000 audio sample points.

Reply:
@IRDonch: Pushed an update to `aclnet.md` according to the above comment from @paulolm.